pallets/xcm-benchmarks/src/fungible/mock.rs

```diff
@@ -0,0 +1,198 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! A mock runtime for XCM benchmarking.
+
+use crate::{fungible as xcm_balances_benchmark, mock::*};
+use frame_benchmarking::BenchmarkError;
+use frame_support::{parameter_types, traits::Everything};
+use sp_core::H256;
+use sp_runtime::{
+	testing::Header,
+	traits::{BlakeTwo256, IdentityLookup},
+	BuildStorage,
+};
+use xcm::latest::prelude::*;
+use xcm_builder::AllowUnpaidExecutionFrom;
+
+type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+type Block = frame_system::mocking::MockBlock<Test>;
+
+// For testing the pallet, we construct a mock runtime.
+frame_support::construct_runtime!(
+	pub enum Test where
+		Block = Block,
+		NodeBlock = Block,
+		UncheckedExtrinsic = UncheckedExtrinsic,
+	{
+		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
+		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
+		XcmBalancesBenchmark: xcm_balances_benchmark::{Pallet},
+	}
+);
+
+parameter_types! {
+	pub const BlockHashCount: u64 = 250;
+	pub BlockWeights: frame_system::limits::BlockWeights =
+		frame_system::limits::BlockWeights::simple_max(1024);
```

Comment:
Why is this so low? Last edit was @shawntabrizi here https://github.com/paritytech/polkadot/pull/2406#discussion_r953663344

Reply:
Would be nice to understand, I will create a ticket with everything that's currently in question.

Reply:
In tests we don't really register any weights in the system pallet (because there's no signed extension to do so), so this number makes no difference afaik. 

You can set it to `.with_sensible_default()` if you want to be pedantic.